gateway/gateway-runtime/Makefile

```
@@ -63,11 +68,35 @@ build: test ## Build Gateway Runtime Docker image using buildx
 		-t $(IMAGE_NAME):latest \
 		--load \
 		.
+	@$(MAKE) extract-policies
 	@rm -rf target
 	@docker run --rm --entrypoint cat $(IMAGE_NAME):$(VERSION) /app/build-manifest.yaml > ../build-manifest.yaml
 	@echo "  ✓ Extracted build-manifest.yaml"
 	@echo "Docker image built successfully: $(IMAGE_NAME):$(VERSION)"
 
+extract-policies: ## Extract controller policy YAMLs from the policy-compiler stage cache to $(POLICIES_EXPORT_DIR)
```

checkmake (0.3.2)

[warning] Target body for "extract-policies" exceeds allowed length of 5 lines (21).

(maxbodylength)